pkg/registry/flowcontrol/prioritylevelconfiguration/storage/storage.go

```diff
@@ -77,15 +73,14 @@ func NewREST(optsGetter generic.RESTOptionsGetter) (*REST, *StatusREST) {
 	return &REST{store}, &StatusREST{store: &statusStore}
 }
 
-// Delete ensures that system priority classes are not deleted.
-func (r *REST) Delete(ctx context.Context, name string, deleteValidation rest.ValidateObjectFunc, options *metav1.DeleteOptions) (runtime.Object, bool, error) {
-	for _, pl := range flowcontrolbootstrap.SystemPriorityLevelConfigurations() {
-		if name == pl.Name {
-			return nil, false, apierrors.NewForbidden(flowcontrol.Resource("prioritylevelconfigurations"), pl.Name, errors.New("this is a system priority level and cannot be deleted"))
-		}
-	}
+// Categories implements the CategoriesProvider interface. Returns a list of categories a resource is part of.
```

Comment:
Does implementing that interface with a method that returns `[]string{""}` produce different behavior than not implementing that interface?

Is it proper to return a list containing the empty string, rather than an empty list?

If we _do_ want to implement that interface, how about a compiler-enforced declaration such as
```
var _ rest.CategoriesProvider = &REST{}
```
?

Reply:
removed